src/Img.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import { createResource } from './createResource';
 
-export const ImgResource = createResource(src => {
+const hashingFn = ({ src, srcSet }) => `${src}${srcset}`;
```

Comment:
Typo here:
`srcset` -> `srcSet`

Reply:
and this is probably a reason that #39 needs to be merged asap. 😄

Reply:
Yikes good catch!  Will take a look at #39 again tomorrow morning.  We also need to set up some PR builds to catch issues like these.

Reply:
@Cryrivers 😬thanks for that catch!

Reply:
@Cryrivers @crosscompile Just in case, I created a PR to fix this typo: https://github.com/palmerhq/the-platform/pull/54

Reply:
@crosscompile sure thing. let me update my branch